.github/workflows/ci.yml

```
@@ -49,3 +49,11 @@ jobs:
         npm install ../../
         cd ../..
     # TODO: Persist the e2e browser results to artifacts
+    # Run Sonar Scan
+    - run: git fetch --unshallow
+      if: ${{ matrix.node-version == '16.x' && runner.os == 'Linux' }}
+    - uses: SonarSource/sonarcloud-github-action@v1.6
+      if: ${{ matrix.node-version == '16.x' && runner.os == 'Linux' }}
```

```suggestion
      if: ${{ matrix.node-version == '16.x' && runner.os == 'Linux' && github.actor != 'dependabot[bot]' }}
```

Unless we add dependabot secrets for sonarcloud